### PR TITLE
Alex/#10 implement localization

### PR DESCRIPTION
**Pull request by aleksandr-efimenko**
- Installed and configured [next-i18next](https://github.com/i18next/next-i18next) library.
- Created 2 locales folders 'en' and 'fr' with 6 JSON files in each that contain all the labels.
- Replaced all the static labels with translated ones on pages and components.
- Added the "interfaceLanguage" field (indentation of other fields is changed) in the User table that contains user settings. Every time the user switches language this change is saved in the DB. Every time the user logs in, the setting is applied.

Related to #10

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -50,14 +50,15 @@ model Session {
...
-    OpenAIModels  OpenAIModels[]
+    id                String         @id @default(cuid())
+    name              String?
+    email             String?        @unique
+    emailVerified     DateTime?
+    image             String?
+    interfaceLanguage String?
+    accounts          Account[]
+    sessions          Session[]
+    OpenAIModels      OpenAIModels[]
```
Comment (cohenoshriconsulting) [inline prisma/schema.prisma:61]: remove this, it is unnecessary complexity

File: public/locales/en/main-page.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "h1": "Welcome to ",
```
Comment (cohenoshriconsulting) [inline public/locales/en/main-page.json:2]: Change the H1 keys to be more meaningful.

File: public/locales/en/main-page.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    "title": "Cybercap | Home",
```
Comment (cohenoshriconsulting) [inline public/locales/en/main-page.json:3]: change title to page_title

File: src/pages/auth/signin/index.tsx [hunk 4]
```diff
@@ -23,28 +33,44 @@ export default function AuthenticationPage() {
...
-              and{" "}
-              <Link
-                href="#"
-                className="underline underline-offset-4 hover:text-primary"
-              >
-                Privacy Policy
-              </Link>
-              .
+              <Trans ns="authentication-page" i18nKey="terms-paragraph">
+                By clicking continue, you agree to our
```
Comment (cohenoshriconsulting) [inline src/pages/auth/signin/index.tsx:40]: [MISSING TRANSLATION]

File: prisma/schema.prisma (hunk 1 above)
Comment (aleksandr-efimenko) [reply prisma/schema.prisma:61]: Removed OpenAiModels column from User model and availableForImageGeneration column from OpenAIModels.

File: src/pages/auth/signin/index.tsx (hunk 4 above)
Comment (aleksandr-efimenko) [reply src/pages/auth/signin/index.tsx:40]: Replaced temp text with [MISSING TRANSLATION]

File: public/locales/en/main-page.json (hunk 3 above)
Comment (aleksandr-efimenko) [reply public/locales/en/main-page.json]: Renamed localization labels

**Pull request merged**